onnx/common/ir_pb_converter.cc

```diff
@@ -54,7 +54,8 @@ static Tensor tensorProtoToTensor(const ONNX_NAMESPACE::TensorProto& tp) {
     case ONNX_NAMESPACE::TensorProto_DataType_FLOAT8E4M3FNUZ:
     case ONNX_NAMESPACE::TensorProto_DataType_FLOAT8E5M2:
     case ONNX_NAMESPACE::TensorProto_DataType_FLOAT8E5M2FNUZ:
-    case ONNX_NAMESPACE::TensorProto_DataType_FLOAT8E8M0:
```

Comment:
Curious why this was removed? Was that the intention here or was this some sort of placeholder? Wouldn't this affect other cases since you're now missinga case for the datatype even though it seems very strange (no mantissa bits at all)